pkg/jx/cmd/delete_application.go

```diff
@@ -155,20 +149,40 @@ func (o *DeleteApplicationOptions) deleteProwApplication(repoService kube.Source
 			}
 		}
 		if o.Org == "" {
-			// Fetch the Org from the stored Custom Resource
-			application, err := repoService.GetSourceRepository(applicationName)
+			// fetch the list of sourcerepositories
+			srList, err := repoService.ListSourceRepositories()
 			if err != nil {
-				return deletedApplications, fmt.Errorf("could not get org for %s. use --org", util.ColorInfo(applicationName))
+				return deletedApplications, fmt.Errorf("error in sourcerepository service %s", err.Error())
+			}
+
+			srObjects := []v1.SourceRepository{}
+			for sr := range srList.Items {
+				if srList.Items[sr].Spec.Repo == applicationName {
+					srObjects = append(srObjects, srList.Items[sr])
+					if len(srObjects) > 1 {
+						return deletedApplications, fmt.Errorf("application %s exists in multiple orgs, use --org to specify the app to delete", util.ColorInfo(applicationName))
```

Comment:
At this point, you could present the user with a filterable list of the multiple orgs and ask them which one to use, rather than returning an error.

Reply:
yeah, i was thinking that would make sense.  Seems fairly easy to do as well.